public/app/features/explore/RawListContainer.tsx

```diff
@@ -0,0 +1,116 @@
+import { css } from '@emotion/css';
+import { cloneDeep } from 'lodash';
+import React from 'react';
+import { VariableSizeList as List } from 'react-window';
+
+import { DataFrame, formattedValueToString } from '@grafana/data/src';
+import { stylesFactory } from '@grafana/ui/src';
+
+import RawList from './RawList';
+
+export type instantQueryRawVirtualizedListData = { Value: string; __name__: string; [index: string]: string };
+
+export interface RawListContainerProps {
+  tableResult: DataFrame;
+}
+
+type instantQueryMetricList = { [index: string]: { [index: string]: instantQueryRawVirtualizedListData } };
+
+const getRawListContainerStyles = stylesFactory(() => {
+  return {
+    wrapper: css`
+      height: 100%;
+      overflow: scroll;
+    `,
+  };
+});
+
+/**
+ * transform dataFrame to instantQueryRawVirtualizedListData
+ * @param dataFrame
+ */
+const getListItemsFromDataFrameNew = (dataFrame: DataFrame): instantQueryRawVirtualizedListData[] => {
```

Comment:
Would say that we can move this function to another file. WDYT?

Reply:
Would be nice to add some tests for it as well.

Reply:
I refactored this out into a new file, but I still need to see if we can get a unit test spun up for this

Reply:
Not the most useful tests, but there is some coverage for this now